programmers/자릿수-더하기.js

```diff
@@ -0,0 +1,8 @@
+function solution(n) {
+    n = String(n).split(``);
```

Comment:
백틱은 문자열 내에 변수가 사용될때 사용하는것이 좋습니다.
저번 리뷰에서 변수가 들어가므로써 ""를 사용한다면 +가 사용되는 상황 같은 경우 말이죠
아래와 같이 문자열내에 변수가 들어가지 않는다면 ' ', " "를 사용하는것이 좋습니다.
```suggestion
    n = String(n).split(``);
```